frank-doc-doclet/src/main/java/org/frankframework/frankdoc/wrapper/FrankClassDoclet.java

```diff
@@ -64,6 +65,19 @@ class FrankClassDoclet implements FrankClass {
 		}
 		AnnotationDesc[] annotationDescs = clazz.annotations();
 		frankAnnotationsByName = FrankDocletUtils.getFrankAnnotationsByName(annotationDescs);
+		initializeVariables();
+	}
+
+	private void initializeVariables() {
+		for(FieldDoc fieldDoc: clazz.fields()) {
+			Object value = fieldDoc.constantValue();
+			if(value != null) {
+				variables.put(fieldDoc.name(), value.toString());
+			}
+		}
+		for(FieldDoc fieldDoc: clazz.enumConstants()) {
+			variables.put(fieldDoc.name(), fieldDoc.name());
```

Comment:
Gaat dit ook goed wanneer het een [documented enum](https://github.com/ibissource/iaf/blob/master/core/src/main/java/nl/nn/adapterframework/doc/DocumentedEnum.java) is?

Reply:
Ik verwacht dat alle implementaties van DocumentedEnum ook gewone enums zijn. Ze worden daarom hetzelfde behandeld als enums die geen DocumentedEnum zijn. De naam van de constante wordt genomen bij het substitueren, niet het label.

Reply:
Maar het label was er nou juist om als waarde te fungeren in plaats van de naam. Je  moet dus als er een label is het label gebruiken.

Reply:
Het gaat bij een documented enum juist om de EnumLabel waarde, dat is immers de waarde die de Frank!Ontwikkelaars moeten configuren.

Reply:
Verwerkt.